Marlin/src/lcd/ultralcd.cpp

```diff
@@ -3965,7 +4046,7 @@ void lcd_quick_feedback(const bool clear_buttons) {
         #if EXTRUDERS == 1
           MENU_MULTIPLIER_ITEM_EDIT(float3, MSG_FILAMENT_LOAD, &filament_change_load_length[0], 0, extrude_maxlength);
         #else // EXTRUDERS > 1
-          MENU_MULTIPLIER_ITEM_EDIT(float3, MSG_FILAMENT_LOAD, &filament_change_load_length[active_extruder], 0, extrude_maxlength);
+          MENU_MULTIPLIER_ITEM_EDIT(float3, MSG_FILAMENT_LOAD " " MSG_ACTIVE, &filament_change_load_length[active_extruder], 0, extrude_maxlength);
```

Comment:
This is not compatible with translations, because different languages use different word orders.